packages/components/core/utils/propsHandler.ts

```diff
@@ -0,0 +1,12 @@
+import { getTarget as getTargetDom } from '@idux/cdk/utils'
+import { Logger } from '../logger'
+
+export function getTarget(target: string | HTMLElement | Window = window): HTMLElement | Window {
+  const targetDom = getTargetDom(target)
+  if (targetDom) {
+    return targetDom
+  } else {
+    Logger.warn(`target does not exist: ${target}, default value are already used: window.`)
+    return window
+  }
+}
```

Comment:
```suggestion
const getTarget = (target: HTMLElement | string | undefined) => {
  if (isHTMLElement(target)) {
    return target
  }
  if (isString(target)) {
    const container = document.querySelector<HTMLElement>(target)
    if (container) {
      return container
    } else {
      Logger.warn(`target does not exist: ${target}, default value are already used: window.`)
    }
  }
  return window
}
```

cdk 里的那个删掉吧，另外记得在 core/utils/index.ts 里 export 一下。